### PR TITLE
ztp: CNF-14638: Cherry pick CLO6 changes to release-4.17

### PR DESCRIPTION
**Pull request by fontivan**
Includes 5 PRs that were merged into main
- https://github.com/openshift-kni/cnf-features-deploy/pull/2028
- https://github.com/openshift-kni/cnf-features-deploy/pull/2042
- https://github.com/openshift-kni/cnf-features-deploy/pull/2046
- https://github.com/openshift-kni/cnf-features-deploy/pull/2047
- https://github.com/openshift-kni/cnf-features-deploy/pull/2049

### CODE REVIEW

**Comment by sabbir-47** [comment]
@fontivan Why are we manually cherrypicking to 4.17 and not using the git workflow?

**Comment by fontivan** [comment]
> @fontivan Why are we manually cherrypicking to 4.17 and not using the git workflow?

I wanted to keep both changes together in one PR to make the backport a bit cleaner

File: ztp/kube-compare-reference/metadata.yaml [hunk 1]
```diff
@@ -5,13 +5,15 @@ Parts:
...
-          - path: required/cluster-logging/ClusterLogging.yaml
           - path: required/cluster-logging/ClusterLogNS.yaml
           - path: required/cluster-logging/ClusterLogOperGroup.yaml
           - path: required/cluster-logging/ClusterLogSubscription.yaml
             config:
               ignore-unspecified-fields: true
           - path: required/cluster-logging/ClusterLogForwarder.yaml
+          - path: required/cluster-logging/ClusterLoggingServiceAccount.yaml
+          - path: required/cluster-logging/ClusterLoggingServiceAccountAuditBinding.yaml
+          - path: required/cluster-logging/ClusterLoggingServiceAccountInfrastructureBinding.yaml
```
Comment (imiller0) [inline ztp/kube-compare-reference/metadata.yaml:16]: Fix paths

Comment (fontivan) [reply ztp/kube-compare-reference/metadata.yaml]: Fixed by cherry picking !2047

**Comment by fontivan** [comment]
cc. @imiller0 @sabbir-47 @lack require an approval and a lgtm for this

**Review by lack — APPROVED** [review]

**Pull request merged**